Makefile

```diff
@@ -308,11 +308,21 @@ define find_all_clojure_files
 $$(comm -23 <(sort <(git ls-files --cached --others --exclude-standard)) <(sort <(git ls-files --deleted)) | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn)
 endef
 
+# Lint Clojure files with clj-kondo.
+#
+# Args:
+#   $1: When equal to true, print warnings.
+define lint_clojure_files
```

Comment:
Does this really need to be a function if we don't reuse it anywhere?

Reply:
Yeah, I started with two targets, ended up with one, so the function is unnecessary.

Reply:
Done https://github.com/status-im/status-mobile/pull/17491/commits/2f2f4dc624995df3ad191bd8eb3ade054c569a4c